frontend/src/js/components/PageViewer/FindInput.tsx

```diff
@@ -1,42 +1,46 @@
-import _ from "lodash";
+import _, { uniq } from 'lodash';
 import React, {
   FC,
   KeyboardEventHandler,
   useCallback,
-  useEffect,
+  useEffect, useMemo,
   useState,
-} from "react";
+} from 'react';
 import DownIcon from "react-icons/lib/md/arrow-downward";
 import UpIcon from "react-icons/lib/md/arrow-upward";
 import styles from "./FindInput.module.css";
+import { HighlightForSearchNavigation } from './model';
+import { Loader } from 'semantic-ui-react';
```

Comment:
I don't love the dependency on Semantic UI but it was quickest way of getting a decent loader icon. We could switch to our own CSS in future PR. I've put this in checklist here: https://trello.com/c/ycfzJavQ/1986-page-viewer-navigate-between-individual-highlights